contrib/src/templates/mod.rs

```diff
@@ -214,27 +242,46 @@ impl Template {
     ///
     /// Returns `Some` if the template could be rendered. Otherwise, returns
     /// `None`. If rendering fails, error output is printed to the console.
+    /// `None` is also returned if a Template fairing has not been attached.
     ///
     /// # Example
     ///
     /// ```rust
     /// use std::collections::HashMap;
+    ///
+    /// extern crate rocket;
+    /// extern crate rocket_contrib;
```

Comment:
We shouldn't need this `extern crate`.

Reply:
Without these, I see a lot of "error[E0432]: unresolved import `rocket_contrib`" or error[E0433]: failed to resolve. Maybe a missing `extern crate rocket;`.

The only success I had was adding both extern crate declarations and wrapping the code in a `fn main()`.

Reply:
Okay, no worries. Let's just move these to the top and hide them with `#`. So the top of the file should look like:

```rust
# extern crate rocket;
# extern crate rocket_contrib;
use std::collections::HashMap;

...
```